### PR TITLE
docs(contribute): improve sentence structure

### PR DESCRIPTION
**Pull request by wizardofhogwarts**
_describe your changes..._
Changed `eg` to `for example`

### CODE REVIEW

**Comment by wizardofhogwarts** [comment]
@EugeneHlushko please review

File: src/content/contribute/index.md [hunk 1]
```diff
@@ -70,6 +71,6 @@ You can also encourage your developers to contribute to the ecosystem by open-so
 
 ### Anyone Else
 
-To anyone else who is interested in helping our mission -- e.g. venture capitalists, government entities, digital agencies, etc. -- we would love for you to work with us, one of the top npm packages, to improve your product! Please don't hesitate to reach out with questions.
```
Comment (EugeneHlushko) [inline src/content/contribute/index.md]: i think thats the same and widely used in the docs (e.g.), why this change?

Comment (wizardofhogwarts) [reply src/content/contribute/index.md]: I think it is more readable with this change. If not required then you can close it. No problem. It is my personal suggestion.

Comment (montogeek) [reply src/content/contribute/index.md]: We use (e.g.) which is widely understandable.
